### PR TITLE
Updated for Synapse 2.8.1

### PR DESCRIPTION
**Pull request by Bonjemus**
yes

### CODE REVIEW

**Comment by AlmightyLks** [comment]
Will look at it later this day, once I've got time

**Comment by Bonjemus** [comment]
Okay.

**Review by AlmightyLks — CHANGES_REQUESTED** [review]
Remove the whole bin folder

**Review by AlmightyLks — CHANGES_REQUESTED** [review]
Why is the Properties folder gone?

File: CustomLoadout/CustomLoadout.csproj [hunk 1]
```diff
@@ -1,87 +1,87 @@
...
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Config\Types\AmmoConfig.cs" />
+    <Compile Include="Config\Types\ItemChance.cs" />
+    <Compile Include="CustomLoadout.cs" />
+    <Compile Include="Config\Types\Loadout.cs" />
+    <Compile Include="Config\PluginConfig.cs" />
+    <Compile Include="Handler\PluginEventHandler.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
```
Comment (AlmightyLks) [inline CustomLoadout/CustomLoadout.csproj:81]: Properties directory gone, but still references to it?

File: CustomLoadout/CustomLoadout.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@ namespace CustomLoadout
         SynapseMajor = 2,
         SynapseMinor = 1,
```
Comment (AlmightyLks) [inline CustomLoadout/CustomLoadout.cs]: Mind updating those numbers to the nuget package version as well?

File: CustomLoadout/CustomLoadout.csproj [hunk 3]
```diff
@@ -1,87 +1,87 @@
...
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
```
Comment (AlmightyLks) [inline CustomLoadout/CustomLoadout.csproj:84]: packages.config gone but still referenced?

**Review by AlmightyLks — CHANGES_REQUESTED** [review]
image

At this state can't be compiled

**Comment by Bonjemus** [comment]
I fixed everything you asked to I think.

**Pull request closed without merging**